### ISSUE TITLE
Release via GitHub with SHA

### ISSUE THREAD

**Issue by raphaklaus** [teamprevue/PreVue#87]
Downloading it via a static AWS link kind of sucks and is not secure. :\

**Comment by al2613** [comment]
Thanks for letting us know! Let me look into this a little more. Were there any specific issues in particular you experienced just so we cover everything?

**Comment by al2613** [comment]
Hey Raphael, 

Thanks for the feedback! The download link is a secure connection so we're not sure what you're referring to exactly? Can you provide us with more specific details?

Thanks,
Annette

**Comment by raphaklaus** [comment]
No, no. I mean, there is no way to ensure the build from AWS is from this source code. You should provide the SHA signature along with the build link from the build system used (Circle, Jenkins, etc).

Search for best security practices using SHA algorithm in open source projects.
